app/controllers/ncr/work_orders_controller.rb

```diff
@@ -4,7 +4,7 @@ class WorkOrdersController < ClientDataController
     MAX_UPLOADS_ON_NEW = 10
 
     def new
-      work_order.approving_official_email = suggested_approver_email
+      @client_data_instance.approving_official = suggested_approver_email
```

Comment:
if you're going to use `@client_data_instance` here instead of the `work_order` method, should we change it everywhere? Inconsistent use may confuse as to whether it was intentional or not.